src/System.Private.CoreLib/shared/System/Resources/ResourceManager.cs

```diff
@@ -178,7 +178,7 @@ private ResourceManager(string baseName, string resourceDir, Type usingResourceS
             BaseNameField = baseName;
 
             _moduleDir = resourceDir;
-            _userResourceSet = usingResourceSet;
+            _userResourceSet = userResourceSet;
```

Comment:
For the `public ResourceManager(string baseName, Assembly assembly, Type usingResourceSet)` below, shouldn't `usingResourceSet` be nullable?